src/Common/src/TypeSystem/Common/TypeSystemContext.cs

```diff
@@ -66,6 +66,13 @@ public virtual ModuleDesc ResolveAssembly(AssemblyName name, bool throwIfNotFoun
             return null;
         }
 
+        public virtual ModuleDesc ResolveModule(ModuleDesc referencingModule, string simpleName, bool throwIfNotFound = true)
```

Comment:
Could you rename the `simpleName` parameter to `fileName`?

Reply:
Oh and can we make this `internal` so that it doesn't pollute intellisense?

Eventually, I assume we're going to add a `GetModule` method to `ModuleDesc` that is going to expose this from a more convenient/expected location (shooting for API surface similar to [Assembly.GetModule](https://msdn.microsoft.com/en-us/library/system.reflection.assembly.getmodule(v=vs.110).aspx)).